modules/serverless-rn-1-22-0.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-22-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.22.0
+
+{ServerlessProductName} 1.22.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-22-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.1.
+* {ServerlessProductName} now uses Knative Eventing 1.1.
+* {ServerlessProductName} now uses Kourier 1.1.
+* {ServerlessProductName} now uses Knative `kn` CLI 1.1.
+* {ServerlessProductName} now uses Knative Kafka 1.1.
+* The `kn func` CLI plug-in now uses `func` 0.23.
+* Knative service specifications now support init containers.
+* The Persistent Volume Claim (PVC) support for Knative services is now available as a Technology Preview.
+* The `knative-serving`, `knative-serving-ingress`, `knative-eventing` and `knative-kafka` system namespaces now have the `knative.openshift.io/part-of: openshift-serverless` label by default.
+* The `Knative Eventing - Kafka Broker/Trigger` dashboard has been added, which allows visualizing Kafka broker and trigger metrics in the web console.
+* The `Knative Eventing - KafkaSink` dashboard has been added, which allows visualizing KafkaSink metrics in the web console.
+* The `Knative Eventing - Broker/Trigger` dashboard is now called `Knative Eventing - Channel-based Broker/Trigger`.
```

Comment:
mentions of dashboards / options in the console should be in bold, not backticks